course-v2/layouts/pages/_markup/render-heading.html

```diff
@@ -0,0 +1,5 @@
+{{ if eq .Level 2}}
+    <h{{ add .Level 1}} id="{{ .Anchor | safeURL }}">{{ .Text | safeHTML }}</h{{ add .Level 1 }}>
+{{ else }}
+    <h{{ .Level }} id="{{ .Anchor | safeURL }}">{{ .Text | safeHTML }}</h{{ .Level }}>
+{{ end }}
```

Comment:
This looks fine but I have a concern/question that will it cause confusion for course authors? Like the course author has set `h2` from studio but it's being rendered as `h3` in Hugo?

Another follow-up question: Is this the correct place to cater to this problem? what if we handle it in studio?
Perhaps if @pdpinch has any thoughts on this?

Reply:
I think I asked for this, but @MAbdurrehman12 makes a good point. What happens if we don't adjust the heading levels? If possible, give me an example of a course in production.

Reply:
@pdpinch Here is the PR preview course in RC with the `course-v2` theme: https://live-qa.ocw.mit.edu/course-v2/courses/15-s21-nuts-and-bolts-of-business-plans-january-iap-2014/

Here is the `course-v2` deploy preview for this PR branch: https://ocw-hugo-themes-course-v2-pr-794--ocw-next.netlify.app/

How font sizes are being set currently:
![image](https://user-images.githubusercontent.com/12089658/185244993-d997591f-aa44-4daf-b9f2-4d1efa44df63.png)

After this PR:
![image](https://user-images.githubusercontent.com/12089658/185245808-dd79ddc7-6251-4436-b511-e26e5259017a.png)

The course title header is now wrapped in an h1 tag, but it isn't doing anything to the font size because the link inside of it has a font specified in it via CSS. As far as the content area though, it seems like we're getting h2 and h3 appropriately sized like you'd expect. Why are we demoting h2 to h3 in the above code? If we do end up doing this, there's no need in specifying `{{ add .Level 1}}` because that will always evaluate to 3 in this case, because the if statement is limited to the level being 2.

Reply:
We've got a couple different requirements, but I can't easily tell if we're meeting them all. 

In order of priority: 

- [ ] every pages should have one and only one H1, for accessibility 
- [ ] additional headers (H2, H3, etc) should follow a logical sequence. In particular, we should avoid skipping any levels. Also for accessibility.
- [ ] Size and color should be correct and consistent, including Instructor Insights pages
- [ ] There should be some logical relationship between the headers users set in Studio (H2, H3, etc) and how they are rendered in the published HTML 
- [ ] We don't want to have to change legacy markdown already in studio. 

Does that help answer your question?

Reply:
@pdpinch Addressed these checkpoints